### DISCUSSION TITLE
AI 畫出來的圖能多美，全靠「提示工程師」！這個未來職業究竟在做什麼？

### DISCUSSION THREAD

**Discussion by peggy0329** [1101-GS4719-C/Python-Community-GS4719-C#929] [科技新知]
文章來源：https://fc.bnext.com.tw/articles/view/2638?utm_source=bn_line&utm_medium=broadcast

先前發過一篇文章是迪士尼透過 AI 新技術，使影片中的中年男子返老還童，而這樣的技術可能使許多化妝師失業。
而今天又看到這篇文，主要説明由 AI 產生的圖片變得奇怪、有趣都是與起初輸入的文字有關，因此「提示工程」出現了！
它是用聰明、準確、時而冗長的文字提示，設定好上下文場景，一步步把 AI 帶進場景，讓它更了解人類意圖，產生最符合期待的結果。
而「提示工程師」不只需具備機器學習、資訊能力，還要對藝術有敏銳度（攝影的光圈原理、色彩學等），才能產生優質的圖片。
讓我不經感嘆，技術的進步，可能造成一個職業的衰退，也可能誕生一個新職業。
而這些正也是跨領域的結合呀

**Comment by yeh-alex** [comment]
其實提示工程師就類似訓練AI模型時下的關鍵字跟訓練方法而已，到頭來還是工程師在做的事情，應該還是無法產生新的職業吧？

**Comment by ching020721** [comment]
可能算工程師需要新的能力吧，除了本質的工程師還需要有藝術細胞

**Comment by CPCpyt** [comment]
我覺得這會讓工程師推升新的高度，
也許日後工程師不僅要會寫程式還要懂藝術跟提是機器人完成工作。

**Comment by StevenWanglolz** [comment]
看來現在學藝術的人也要具備資訊能力和瞭解機器學習了嗎?

**Comment by BlahFish1006** [comment]
現代的學生不管是什麼科系的好像都需要具備一定的程式能力，壓力好大呀......
但在有程式能力的基礎下，確實能幫助我們實現更多不一樣的可能性，有好有壞。

**Comment by ghost** [comment]
很有趣的工作，感覺這種提示工程師很像是音樂家在寫譜，目標都是找出符合的搭配組合，說不定未來藝術領域會出現提示派畫家😂

**Comment by dinosaur1009** [comment]
感覺現在的科技都是要結合多方領域，蠻強調跨領域合作的，感覺是個趨勢！

**Comment by Kotori0501** [comment]
這感覺很像機器學習中的特徵工程，
但我覺得有時後在AI繪圖中如果輸出很多的關鍵字可能會造成反效果，而且我認為還是自己畫出屬於自己的風格更重要。
這項技術雖然很方便，但我還是覺得最原始的方法，還有漫畫家畫的圖還是比AI還要自然純樸，所以我不太認為會有新職業產生

**Comment by Tuoresa** [comment]
感覺很神奇！
人工智慧真的能做到的事情無可限量欸
看來工程師要做到的事情越多
要具備的能力也需要更加多元和全面了

但總覺得透過筆觸真實展現出來的作品
才會隱含著作者自身的氣息與故事，就算有瑕疵也無妨
感覺那才能真正展現出藝術的獨特性與珍貴
而不是只要「美」就夠了

**Comment by ru0372572** [comment]
工程師的工作變成跨領域的一條出路了嗎，但還是覺得藝術作品須要有故事的人來做才會有有價值的感覺（感覺很古板哈哈）

**Comment by yuxinlin93** [comment]
最近看到那一些AI的產徒都在想，該不會以後藝術學校要多一個科目是用程式碼畫圖，藝術家也要會操作AI來畫出他們要的圖，也太累了吧

**Comment by Jzd314** [comment]
通識課中有實際用AI畫過圖，畫出來的圖精緻到我用描的都描不出來

**Comment by harrylee1971** [comment]
可能也只有設計AI本身的工程師不會失業了吧，醫師讀七年到最後也還是有可能被AI取代掉
